### ISSUE TITLE
internationalizedArrayText crashes Studio with Error: Cannot read properties of undefined (reading 'default')

### ISSUE THREAD

**Issue by ervibern** [sanity-io/sanity-plugin-internationalized-array#54]
**Describe the bug**

I created several internationalized fields in my scheme. The `internationalizedArrayString` works well while `internationalizedArrayText` results in a Studio crash.
```
defineField({
      name: 'title',
      title: 'Title',
      type: 'internationalizedArrayString',
      validation: (rule) => rule.required(),
    }),
```
```
defineField({
      name: 'text',
      title: 'Text',
      type: 'internationalizedArrayText',
    }),
```
Error: Cannot read properties of undefined (reading 'default')

In sanity.config.js, I added a plugin, like that:
```
internationalizedArray({
      languages: [
        {id: 'en', title: 'English'},
        {id: 'lt', title: 'Lithuanian'},
      ],
      defaultLanguages: ['en'],
      fieldTypes: ['string', 'text'],
    }),
```

**To Reproduce**

Setup a scheme as above and run `npm run dev` to open localhost.

**Expected behavior**

Expect both string and text fields to be internationalized.

**Screenshots**

<img width="1131" alt="image" src="https://github.com/sanity-io/sanity-plugin-internationalized-array/assets/58375689/6fc7cdd5-9690-451e-bd1d-1078a05e34e9">

**Which versions of Sanity are you using?**

@sanity/cli (global)          3.23.4 (up to date)
@sanity/eslint-config-studio   3.0.1 (up to date)
@sanity/vision                3.23.4 (up to date)
sanity                        3.23.4 (up to date)

**What operating system are you using?**

MacOS Ventura 13.4

**Which versions of Node.js / npm are you running?**

npm: 10.2.3
node: v20.10.0

**Additional context**

"sanity-plugin-internationalized-array": "^1.10.3",

**Comment by thomasKn** [comment]
Same issue here, I think this is related to the styled-components missmatch issue (#52)

**Comment by plcdnl** [comment]
is there any news about it? It would be very useful to be able to use it

**Comment by robinpyon** [comment]
Thanks for reporting! A fix for this issue is forthcoming, and should be out in the next Sanity studio release.

**Comment by michaelburtonray** [comment]
> is there any news about it? It would be very useful to be able to use it

@plcdnl The latest version of the Studio fixes this bug! v3.24.1 https://github.com/sanity-io/sanity/releases
